talks/index.html

```diff
@@ -11,7 +11,12 @@
             font-weight: 400;
             line-height: 1.5;
         }
+        e-kursy-it-embed {
+          height: 370px;
```

Comment:
1. please add: `width: 480px;` to make all videos the same width.
2. I tryied it locally, but the movie doesn't work - after "Play" click it's frozen and no sound. Could you check this please?
Only timeframe works.
3. It would be nice to have a possibility to reditect to your page - like in YouTube videos - you can click YT logo to open a movie in new tab.